Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -270,7 +271,7 @@ open class AVFoundationPlayback: Playback {
     }
     #endif
 
-    @objc internal func addObservers() {
+    @objc internal func setupObservers() {
```

Comment:
I think we can omit the `internal` access level.

Reply:
Done